cedar-policy-core/src/evaluator/err.rs

```diff
@@ -43,8 +43,8 @@ pub enum EvaluationError {
 
     /// Tried to get an attribute of a (non-entity) record, but that record
     /// didn't have that attribute
-    #[error("record does not have the attribute: {0}")]
-    RecordAttrDoesNotExist(SmolStr),
+    #[error("record does not have the attribute: {0}. Available attrs: {1:?}")]
```

Comment:
Consistency nit:
```suggestion
    #[error("record does not have the attribute: {0}. Available attributes: {1:?}")]
```
Also, what does the `{1:?}` syntax mean? What does this print if the list of available attributes is empty? Possibly we want a separate function (like `pretty_type_error` below) that prints something like "record has no attributes" in this case.